pocs/observatory.py

```diff
@@ -199,43 +198,82 @@ def get_observation(self, *args, **kwargs):
 
         return self.current_observation
 
-    def cleanup_observations(self):
+    def cleanup_observations(self, timeout=600):
```

Comment:
FWIW, cleanup doesn't seem the correct term. The main focus of this method seems to be uploading.

Reply:
The cleanup is the main purpose of this method, namely to remove old files and to fpack the FITS files. The making of a timelapse and uploading are optional to that. Cleanup always has to happen, uploading does not.